HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -22,13 +22,15 @@ public record LocalAuthorityDetails(
     private enum IncomeThreshold
     {
         _31000,
-        _34500
+        _34500,
```

Comment:
I think we can get rid of these two from here now, to avoid any future confusion